sklearn/preprocessing/_encoders.py

```diff
@@ -155,7 +155,9 @@ class OneHotEncoder(_BaseEncoder):
     strings, denoting the values taken on by categorical (discrete) features.
     The features are encoded using a one-hot (aka 'one-of-K' or 'dummy')
     encoding scheme. This creates a binary column for each category and
-    returns a sparse matrix or dense array.
+    returns a sparse matrix or dense array (depending on the sparse parameter,
+    which by defaut is True and gives sparse matrix and when false dense array
```

Comment:
this seems redundant, we explain this parameter below (See parameters section).
``(depending on the sparse parameter)`` seems enough.

Reply:
@qinhanmin2014 Done